users/models.py

```diff
@@ -68,3 +69,34 @@ class User(AbstractBaseUser, PermissionsMixin):
 
     def __str__(self):
         return self.email
+
+
+class Owner(models.Model):
+    user = models.OneToOneField(AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
```

Comment:
Are you planning on adding additional fields to Owner? If not, this model has no purpose.

Reply:
I'm just waiting for an answer to this question before approving.

Reply:
According to yesterday's session, I understand from the group that we are planning on adding additional fields to Owner.
But now that I reconsider it, I think it might really be better to give up the Owner class.
I would like to hear more from the other teammates before I take action and delete it.
@Micha546 @AmitAharoni @DanielMy777 @nadavsu

Reply:
Lets remove this model, I don't see a reason to keep it other than organization.

Reply:
I also think that we should remove the Owner class and only use Apartment.
We don't really have a use for the Owner right now...

Reply:
Just keep the Aprtment class, It will be enough

Reply:
Ok, so I will remove the Owner and change the Apartment 'owner' field.